src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,51 @@
+.menu {
+  height: 100vh;
+
+  font-weight: 700;
+  color: $main-white;
+  background-color: $accent-main;
+
+  overflow: auto;
+
+  &__top {
+    margin-bottom: 40px;
+
+    position: sticky; /* to keep cross always visible */
+    top: 0;
+    z-index: 1;
+    background-color: $accent-main;
+  }
+
+  &__content {
+    @include content-padding-inline;
+  }
+
+  &__nav {
+    margin-bottom: 32px;
+  }
+
+  &__contact {
+    display: flex;
+    flex-direction: column;
+
+    gap: 8px;
+    margin-bottom: 16px;
+  }
+
+  &__phone-number {
+    font-size: 16px;
+    line-height: 27px;
+    color: inherit;
+  }
+
+  &__call-to-order {
+    width: fit-content;
+    padding-bottom: 7px;
+    border-bottom: 1px solid $main-white;
+
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 2px;
+    color: inherit;
```

Comment:
you don't need this style

Reply:
Can't understand why color: inherit there is redundant.
It is class used in <a> tag, so without this style my color on links in menu reverts back to standart blue of <a> tag.
In new commit i moved color: inherit to style <a> tag, so all links will inherit father color, but i don't really sure if it's a right solution.
Can you explain a llitle bit further why i don't need this style?

Reply:
Using `inherit` is not a good idea, because we can get a style side effects. 
Our color is based on the parent element. If in the future change the color in a parent element, also, changes the color child element. ( this is a side effect). 
In this case, it is better to use those colors that should definitely be here, so that their value needs to be changed additionally and the developer does not miss this moment and does not allow a side effect.